source/common/router/router_ratelimit.cc

```diff
@@ -232,6 +257,20 @@ void RateLimitPolicyEntryImpl::populateDescriptors(
   }
 }
 
+void RateLimitPolicyEntryImpl::populateLocalDescriptors(
+    const Router::RouteEntry& route, std::vector<RateLimit::LocalDescriptor>& descriptors,
+    const std::string& local_service_cluster, const Http::HeaderMap& headers,
+    const Network::Address::Instance& remote_address,
+    const envoy::config::core::v3::Metadata* dynamic_metadata) const {
+  RateLimit::LocalDescriptor descriptor;
+  descriptor.token_bucket_ = {};
```

Comment:
nit: not needed

Reply:
Just adding why I did this: Added this because otherwise comparisons in test were becoming difficult and it was good to initialize it to an empty value.

Reply:
This should be done by the constructor then.

Reply:
Done.